.evergreen/debug-core-evergreen.sh

```diff
@@ -7,3 +7,11 @@ for i in *.core; do
    echo $i
    echo "backtrace full" | gdb -q ./src/libmongoc/test-libmongoc $i
 done
+
+# If there is still a test-libmongoc process running (perhaps due to
+# deadlock, or very slow test) attach a debugger and print stacks.
+TEST_LIBMONGOC_PID="$(pgrep test-libmongoc)"
+if [ -n "$TEST_LIBMONGOC_PID" ]; then
+   echo "test-libmongoc processes still running with PID=$TEST_LIBMONGOC_PID"
+   echo "backtrace full" | gdb -q -p $TEST_LIBMONGOC_PID
+fi
```

Comment:
This hasn't been used on evergreen, but I'm adding this as a "just-in-case" a deadlock appears. I did introduce a deadlock locally in the integration tests (by joining on a thread while holding a lock that the thread needed to proceed) and attaching lldb and printing every threads stack trace helped me diagnose.